filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -277,27 +293,79 @@ class DrsPathBuilderSpec extends TestKitSuite with AnyFlatSpecLike with Matchers
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false),
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "an absolute path without a host",
       path = s"drs://blah",
       normalize = false,
-      pathAsString = s"drs://blah/",
-      pathWithoutScheme = s"blah/",
+      pathAsString = s"drs://blah",
+      pathWithoutScheme = s"blah",
       parent = null,
       getParent = null,
-      root = s"drs://blah/",
+      root = s"drs://blah",
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false)
+      isAbsolute = false,
+    ),
+
+    // No spec says this is illegal... so pass it to Martha's various GCFs JIC
+    GoodPath(
+      description = "a bucketless path",
+      path = s"drs://",
+      normalize = false,
+      pathAsString = s"drs://",
+      pathWithoutScheme = "",
+      parent = null,
+      getParent = null,
+      root = s"drs://",
+      name = "",
+      getFileName = null,
+      getNameCount = 1,
+      isAbsolute = false,
+    ),
+
+    // https://docs.google.com/document/d/1Wf4enSGOEXD5_AE-uzLoYqjIp5MnePbZ6kYTVFp1WoM/edit
+    GoodPath(
+      description = "a path with a query string",
+      path = "drs://drs.data.humancellatlas.org/8aca942c-17f7-4e34-b8fd-3c12e50f9291?version=2019-07-04T151444.185805Z",
+      normalize = false,
+      pathAsString =
+        "drs://drs.data.humancellatlas.org/8aca942c-17f7-4e34-b8fd-3c12e50f9291?version=2019-07-04T151444.185805Z",
+      pathWithoutScheme =
+        "drs.data.humancellatlas.org/8aca942c-17f7-4e34-b8fd-3c12e50f9291?version=2019-07-04T151444.185805Z",
+      parent = null,
+      getParent = null,
+      root =
+        "drs://drs.data.humancellatlas.org/8aca942c-17f7-4e34-b8fd-3c12e50f9291?version=2019-07-04T151444.185805Z",
```

Comment:
Do the DRS spec writers realize that they're invalidating "query-string" like syntax in DRS urls?

Reply:
Heh, dunno. If it's a problem, we'll submit yet another PR just like this one to fix it.